<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="ButtonFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>741</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="newButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>8</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="contButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="pauseButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="restButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Restart</string>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="WavePlot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="InfoTags">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>239</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="BasictTag">
     <attribute name="title">
      <string>Basics</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="ThreadTag">
     <attribute name="title">
      <string>Threads</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>321</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>559</width>
        <height>319</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
